_caseStudies/2022-10-07-oss-usage-east-africa.md

```diff
@@ -0,0 +1,92 @@
+- There is an insufficient number of people within the organization who are well-versed with the tools and there is also limited skill transfer or capacity building within the organization..
+
+*Monitoring and evaluation*
+- Some of the tools are very complicated for beginners or non-MERL experts, and some of them do not have dashboard features to show quick analyses or descriptive statistics for indicators being analyzed.
+
+Seven of the 15 respondents noted that they have received training for using the tools that they use in their organization, primarily in-house. Most respondents think that people responsible for the above tasks have a high level of proficiency in using the tools mentioned. Both international and national staff were involved in the entire MERL technology workflow, but international staff were more likely to be involved in data analysis and report writing.
+
+The main factors they consider in selecting tools to use to complete their MERL tasks include ease of use and industry standards. Ease of use refers to how intuitive the tools are, how easy they are to configure and navigate, and whether or not tools require formal training. Moreover, one of the respondents noted the interoperability of tools to allow for collaborative work. One of the respondents mentioned using OSS, “[Using] open source so that everyone working together can have access." As mentioned in the table above, standardization of tools within the industry or sector is an important factor to consider because it implies that the tools are commonly used and therefore there is a wealth of technical support available.
+
+Among the six respondents that identified the decision-maker on which tools to use to perform MERL tasks, four said that their director, team leader, or the head of the organization made the decision. This is consistent with findings from research conducted during the [design phase of the MERL Center](https://merltech.org/designing-a-merl-github-center/). The study found that decision-making on technology use in the social sector does not lie with information technology (IT teams) but rather with management teams in-charge of making budgetary decisions.
+
+##### Familiarity with OSS
+The series of questions under this section aims to determine if respondents know what tools are considered OSS and why they are considered as such. Four of five respondents said that they were either fairly or very familiar with OSS. Respondents were asked what they knew about OSS - (1) the source code is publicly available, (2) users can contribute to the development of the software, and (3) there are little to no financial costs involved in using the software, but data hosting, protection, and sharing could be challenging. One respondent gave examples of OSS that they know about such as ODK and QGIS.
```

Comment:
point 3: Free in "Free Open source" usually refer to the 4 freedoms as compared to cost, thus sometimes the phrase Libre is used instead of Free. "Free as in freedom and free beer" We have to make it clear that free does not refer to the cost.